src/main/java/org/testng/internal/annotations/AnnotationHelper.java

```diff
@@ -217,7 +219,7 @@ private static void finishInitialize(ConfigurationAnnotation result, IConfigurat
 //    for (Class<?> cls : classes) {
         while (null != cls) {
           boolean hasClassAnnotation = isAnnotationPresent(annotationFinder, cls, annotationClass);
-          Method[] methods = cls.getDeclaredMethods();
+          Method[] methods = getDeclaredMethods(cls);
```

Comment:
I think we should find another name. The difference between `cls.getDeclaredMethods()` and `getDeclaredMethods(cls)` is not explicit enough.

I'd prefer something like (pseudo code): `cls.getDeclaredMethods() + getDefaultMethods(cls)`
Or `add(cls.getDeclaredMethods() + getDefaultMethods(cls))`
Or just `getAllMethods()`